package.json

```diff
@@ -21,11 +21,11 @@
   },
   "main": "index.js",
   "dependencies": {
-    "fibers": "^1.0.7",
+    "fibers": "^3.1.0",
     "meteor-promise": "^0.8.2",
     "underscore": "^1.8.3",
-    "wdio-sync": "^0.6.6",
-    "webdriverio": "^4.4.0",
-    "xolvio-fiber-utils": "^2.0.3"
+    "wdio-sync": "^0.7.0",
+    "webdriverio": "^4.14.0",
+    "xolvio-fiber-utils": "github:sirphoenix/fiber-utils"
```

Comment:
this will have to change to the new xolvio-fiber-utils after this is merged and published: 
https://github.com/xolvio/fiber-utils/pull/2

@samhatoum

Reply:
if you merge the pull requestr on fiber-utils this should work now